backend/spacewalk-backend.spec

```diff
@@ -334,11 +339,9 @@ sed -i 's/#DOCUMENTROOT#/\/srv\/www\/htdocs/' $RPM_BUILD_ROOT%{rhnconfigdefaults
 install -m 755 satellite_tools/mgr-update-pkg-extra-tags $RPM_BUILD_ROOT%{_prefix}/lib/susemanager/bin/
 
 ## Install Zypper plugins only on SUSE machines
-%if 0%{?suse_version}
 install -Dd -m 0750 % $RPM_BUILD_ROOT%{_prefix}/lib/zypp/plugins/urlresolver
 %{__install} satellite_tools/spacewalk-uln-resolver $RPM_BUILD_ROOT%{_prefix}/lib/zypp/plugins/urlresolver/spacewalk-uln-resolver
 %{__install} satellite_tools/spacewalk-extra-http-headers $RPM_BUILD_ROOT%{_prefix}/lib/zypp/plugins/urlresolver/spacewalk-extra-http-headers
-%endif
```

Comment:
Are you using zypper on CentOS?

Reply:
No, I have removed the if statement for simplicity/cleanness. Otherwise we would require additional if statements in the %files section.
I guess it is also an ideology question:

1. Remove all code that is not available in other distributions and only keep the used code?
2. Keep all code and only remove the failing code?

Number 1. is nicer but 2. looks more realistic to me and is easier to keep consistent.

Reply:
Fine with me. I will merge